WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/OrderCustomerSection.swift

```diff
@@ -26,23 +28,30 @@ struct OrderCustomerSection: View {
 
                     if viewModel.isDataAvailable {
                         Button(Localization.editButton) {
-                            showAddressForm.toggle()
+                            showCustomerSummary.toggle()
                         }
                         .buttonStyle(LinkButtonStyle())
                         .fixedSize(horizontal: true, vertical: true)
                         .padding(.top, -Layout.linkButtonTopPadding) // remove padding to align button title to the top
                         .padding(.trailing, -Layout.linkButtonTrailingPadding) // remove padding to align button title to the side
+                        .sheet(isPresented: $showCustomerSummary) {
+                            NavigationView {
+                                OrderCustomerSummary(viewModel: viewModel)
+                            }
+                        }
```

Comment:
nit: What do you think about moving this call to `.sheet(isPresented: $showCustomerSummary)` down to where we call `.sheet(isPresented: $showAddressForm)`? I don't think we have any conventions about that in our project yet but it might be nice to place them somewhere consistent for readability.